src/app/router/Welcome/Welcome.tsx

```diff
@@ -73,10 +76,10 @@ function App() {
   return (
     <div>
       {process.env.NODE_ENV === "development" && <DevMenu />}
-
+      <LocaleSwitcher />
       <div className="relative max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
         <div className="text-center font-serif font-medium text-2xl pt-7 pb-3 dark:text-white">
-          <p>The power of lightning in your browser</p>
+          <p>{t("heading")}</p>
```

Comment:
what is the convention for naming these keys? 
i18n automatically nests these keys under the component names? (do we need to document this for translators?)

Reply:
I think it's a combination with `useTranslation(["welcome"])` in line 53.  
This is a short way of writing `welcome.heading`.